### PR TITLE
Adding example: Concat and Repeat

### PR DESCRIPTION
**Pull request by ydlamba**
**Adding example: Concat and Repeat**

- All examples of concat and repeat from docs are added to the gallery.

- Link to the interactive version of SPLOM is added to docs.

Solves #3469.

### CODE REVIEW

File: _data/examples.json [hunk 1]
```diff
@@ -244,6 +244,16 @@
...
+    },
+    {
+      "name": "vconcat_weather",
+      "title": "Vertically concatenated charts that show precipitation in Seattle",
+      "description": "Two vertically concatenated charts that show a histogram of precipitation in Seattle and the relationship between min and max temperature."
+    },
+    {
+      "name": "repeat_histogram",
+      "title": "Horizontally repeated charts",
+      "description": "Horizontally repeated charts that show the histograms of different parameter of cars in different countries."
```
Comment (kanitw) [inline _data/examples.json]: parameters

File: site/docs/composition/repeat.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ Repeat can be used to create a scatterplot matrix (SPLOM), where each cell shows
 
 <span class="vl-example" data-name="repeat_splom_iris"></span>
 
+You can also check the [interactive version](https://vega.github.io/vega-lite/examples/interactive_splom.html) of this example.
```
Comment (kanitw) [inline site/docs/composition/repeat.md]: "You can also check the interactive SPLOM example".   

This example is 4x4, the interactive one is 3x3, so not exactly the same.

**Comment by ydlamba** [comment]
@kanitw Updates done.

**Comment by kanitw** [comment]
Thanks for helping :)

**Comment by kanitw** [comment]
FYI,  `solves` is not a valid keyword for closing issues.   Here are [ones that work](https://help.github.com/articles/closing-issues-using-keywords/).

**Comment by kanitw** [comment]
I normally use `fix`

**Pull request merged**